tofhir-server/src/main/scala/io/tofhir/server/endpoint/FhirDefinitionsEndpoint.scala

```diff
@@ -41,13 +42,42 @@ class FhirDefinitionsEndpoint(fhirDefinitionsConfig: FhirDefinitionsConfig) exte
           }
         }
       }
+    } ~ pathPrefix(SEGMENT_VALIDATE) {
```

Comment:
I think we can put it under pathPrefix(SEGMENT_FHIR_DEFINITIONS). WDYT @YemreGurses  ?

Reply:
We decided to leave it as it is